projects/demo/src/modules/components/pdf-viewer/index.html

```diff
@@ -6,14 +6,8 @@
     <ng-template pageTab>
         <p>Custom dialog to preview PDF</p>
 
-        <tui-notification status="warning">
-            Keep in mind mobile devices do not support displaying PDFs in iframe, so you need to rely on
```

Comment:
I am not sure that we should remove this warning from the top of the documentation page.

This issue https://github.com/taiga-family/taiga-ui/issues/8182 is create to just update text of this warning – provide the full list (or link to this information) which mobile devices supports pdf inside iframe and which ones not.